engine/engine.go

```diff
@@ -518,10 +519,17 @@ func createEngine(terragruntOptions *options.TerragruntOptions) (*proto.EngineCl
 
 	terragruntOptions.Logger.Debugf("Creating engine %s", localEnginePath)
 
+	engineLogLevel := os.Getenv(EngineLogLevelEnv)
```

Comment:
Consider whether you need special behavior to handle this:
https://github.com/gruntwork-io/terragrunt/blob/58e2c20ed2c50b4bdc3c278a01bc447fab650d78/cli/commands/flags.go#L352-L361

We might want to suppress logging in the engine too when it's disabled in Terragrunt.

Reply:
Added setting of "off" log level